stickers.js

```diff
@@ -3,14 +3,19 @@ var stickers = [{
 
   },
   create: function (sticker) {
+    var allStickers = Object.values(this.getAll());
+    allStickers.push({
```

Comment:
Object.values transforming object into the array, and pushing new sticker object into the array of stickers. somehow I think it is not working and will try re-do it again